apps/frontend/src/app/poll/choose-events/choose-events.component.ts

```diff
@@ -202,13 +206,23 @@ export class ChooseEventsComponent implements OnInit {
     } else if (maxParticipants && this.participants.length >= maxParticipants) {
       this.closedReason = 'This poll has reached it\'s maximum number of participants. You can no longer submit your vote.';
       this.showResults = true;
+    } else if (this.poll?.settings?.showResult === ShowResultOptions.NEVER as ShowResultOptions) {
```

Comment:
Why `as`?

Reply:
I had some issues there last time while coding, because it wasn't recognized as ShowResultOptions and forgot to remove it. Fixed in a168337.